### PR TITLE
Sentry Catalog #404

### LINKED ISSUE #404: Sentry Catalog
Hi,
   It seems like sentry_email_password has been duplicated with types: password & string.

https://github.com/rancher/community-catalog/blob/master/templates/sentry/0/rancher-compose.yml#L49

https://github.com/rancher/community-catalog/blob/master/templates/sentry/0/rancher-compose.yml#L55

Also, will it  be possible to update this to latest version?

Thanks

### PR DESCRIPTION
**Pull request by wahmedswl**
Hi,
   This PR will fix #404.

### CODE REVIEW

File: templates/sentry/1/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    SENTRY_SECRET_KEY: ${sentry_secret_key}
+    SENTRY_SERVER_EMAIL: ${sentry_server_email}
+  log_driver: ''
+  labels:
+    io.rancher.container.pull_image: always
+  tty: true
+  command:
+  - /bin/bash
+  - -c
+  - sentry upgrade --noinput && sentry createuser --email ${sentry_inital_user_email} --password ${sentry_inital_user_password} --superuser && /entrypoint.sh run web || /entrypoint.sh run web
```
Comment (greg0ire) [inline templates/sentry/1/docker-compose.yml]: - `s/inital/initial`
- why not create the user directly in the entrypoint?

File: templates/sentry/1/rancher-compose.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      type: "string"
+      required: true
+      label: "SENTRY_INITAL_USER_EMAIL"
+      description: "The initial superuser email"
+
+    - variable: "sentry_inital_user_password"
+      type: "password"
+      required: true
+      label: "SENTRY_INITAL_USER_PASSWORD"
+      description: "The initial superuser password. Please use a simple intial password and change it afterwards in the Sentry interface"
```
Comment (greg0ire) [inline templates/sentry/1/rancher-compose.yml]: "intial" => "initial"

**Comment by greg0ire** [comment]
Here's a stack I built today, I have no time to contribute it properly, but maybe you can? : 

```
version: '2'
volumes:
  sentry-pgdata:
    external: true
    driver: 'null'
  sentry-redis:
    external: true
    driver: 'null'
services:
  sentry-database:
    image: postgres:9.6-alpine
    environment:
      PGDATA: /var/lib/postgresql/data
      POSTGRES_DATABASE: sentry
      POSTGRES_PASSWORD: redacted
      POSTGRES_USER: sentry
    stdin_open: true
    tty: true
    volumes_from:
    - sentry-database-datavolume
    labels:
      io.rancher.scheduler.affinity:host_label: sentry.db=true
      io.rancher.sidekicks: sentry-database-datavolume
      io.rancher.container.hostname_override: container_name
      io.rancher.container.pull_image: always
  sentry-cron:
    image: sentry:8.16.0
    environment:
      SENTRY_EMAIL_HOST: redacted
      SENTRY_SECRET_KEY: redacted
      SENTRY_SERVER_EMAIL: redacted
      SENTRY_POSTGRES_HOST: postgres
      SENTRY_DB_NAME: sentry
      SENTRY_DB_PASSWORD: redacted
      SENTRY_DB_USER: sentry
    stdin_open: true
    tty: true
    links:
    - sentry-redis:redis
    - sentry-database:postgres
    command:
    - run
    - cron
    labels:
      io.rancher.container.pull_image: always
  sentry-database-datavolume:
    image: alpine
    stdin_open: true
    entrypoint:
    - /bin/true
    volumes:
    - sentry-pgdata:/var/lib/postgresql/data
    tty: true
    labels:
      io.rancher.container.pull_image: always
      io.rancher.container.start_once: 'true'
  sentry-worker:
    image: sentry:8.16.0
    environment:
      SENTRY_EMAIL_HOST: redacted
      SENTRY_SECRET_KEY: redacted
      SENTRY_SERVER_EMAIL: redacted
      SENTRY_POSTGRES_HOST: postgres
      SENTRY_DB_NAME: sentry
      SENTRY_DB_PASSWORD: redacted
      SENTRY_DB_USER: sentry
    stdin_open: true
    tty: true
    links:
    - sentry-redis:redis
    - sentry-database:postgres
    command:
    - run
    - worker
    labels:
      io.rancher.scheduler.affinity:host_label: sentry.worker=true
      io.rancher.container.pull_image: always
      io.rancher.scheduler.global: 'true'
  sentry-redis:
    image: redis:3.2-alpine
    stdin_open: true
    volumes:
    - sentry-redis:/data
    tty: true
    labels:
      io.rancher.container.pull_image: always
      io.rancher.container.hostname_override: container_name
  sentry:
    image: sentry:8.16.0
    environment:
      SENTRY_EMAIL_HOST: redacted
      SENTRY_SECRET_KEY: redacted
      SENTRY_SERVER_EMAIL: redacted
      SENTRY_POSTGRES_HOST: postgres
      SENTRY_DB_NAME: sentry
      SENTRY_DB_PASSWORD: redacted
      SENTRY_DB_USER: sentry
    stdin_open: true
    tty: true
    links:
    - sentry-redis:redis
    - sentry-database:postgres
    command:
    - /bin/bash
    - -c
    - sentry upgrade --noinput && sentry createuser --email redacted --password redacted --superuser && /entrypoint.sh run web || /entrypoint.sh run web
    labels:
      io.rancher.container.pull_image: always

```

**Comment by rawmind0** [comment]
Closed due to inactivity, superseded by https://github.com/rancher/community-catalog/pull/544

**Pull request closed without merging**